pkgs/tools/misc/djent/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "djent";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "dj-on-github";
+    repo = "djent";
+    rev = "${version}";
+    sha256 = "sha256-inMh7l/6LlrVnIin+L+fj+4Lchk0Xvt09ngVrCuvphE=";
+  };
+
+  buildInputs = [ mpfr ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp djent $out/bin
+  '';
+
+  meta = {
+    homepage = "http://www.deadhat.com/";
+    description = "A reimplementation of the Fourmilab/John Walker random number test program ent with several improvements";
```

Comment:
nit: Use multiline strings `'' ''` and wrap the string at a sensible point

Reply:
Also done